module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -155,7 +155,7 @@ public function homeAction()
                     if (
                         $this->params()->fromPost('processLogin')
                         && $this->inLightbox()
-                        && empty($this->getFollowupUrl())
+                        && empty($this->hasFollowupUrl())
```

Comment:
I don't think you need an empty check here since `hasFollowupUrl` returns a boolean.

Reply:
Fixed